exonum/src/storage/map_index.rs

```diff
@@ -190,7 +194,7 @@ where
         self.base.contains(key)
     }
 
-    /// Returns an iterator over the entries of the map in ascending order. The iterator element
+    /// Returns an iterator over the entries of a map in ascending order. The iterator element
```

Comment:
Nit: I'm not sure the indefinite article is the best choice here; we're talking about a specific map, for which the method is invoked.